frontend/uddhar-frontend/src/shared/components/ScrollableEvent.jsx

```
@@ -23,20 +24,28 @@ const ScrollableEvent = ({
-        </button>
-      </div>
+      {/* heading and disaster creation for coordinator*/}
+      {isCoordinator && (
+        <div className="flex justify-between align-middle">
+          <p className="text-3xl">{heading}</p>
+          <button
+            onClick={() => {
+              window.location.href = "/create-a-event";
+            }}
+            className="bg-[#FF0800] blinking px-10 py-2 rounded cursor-pointer font-bold hover:-translate-y-0.5 hover:bg-[#FF0800] hover:text-white hover:font-extrabold"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)